src/components/Dashboard/FilingHistoryList/StaffNotation.vue

```diff
@@ -0,0 +1,177 @@
+                            :class="{active: expand}"
+                            @click="expand = !expand" >
+                            <v-icon>mdi-menu-down</v-icon>
+                        </v-btn>
+                    </span>
+                </template>
+                <v-list dense>
+                    <v-list-item-group color="primary">
+                        <v-list-item @click="showRegistrarsNotationDialog();expand = !expand">
+                            <v-list-item-title
+                                class="file-correction-item"
+                            >
+                                <span class="app-action">Add Registrar's Notation</span>
+                            </v-list-item-title>
+                        </v-list-item>
+                        <v-list-item @click="showRegistrarsOrderDialog();expand = !expand">
+                            <v-list-item-title
+                                class="add-detail-comment-item"
+                            >
+                                <span class="app-action">Add Registrar's Order</span>
+                            </v-list-item-title>
+                        </v-list-item>
+                        <v-list-item @click="showCourtOrderDialog();expand = !expand">
+                            <v-list-item-title
+                                class="add-detail-comment-item"
+                            >
+                                <span class="app-action">Add Court Order</span>
+                            </v-list-item-title>
+                        </v-list-item>
+                    </v-list-item-group>
+                </v-list>
+            </v-menu>
+        </div>
+    </div>
+</template>
+<script lang="ts">
+import { Component, Prop, Emit, Vue } from 'vue-property-decorator'
+
+// Dialog
+import { AddStaffNotationDialog } from '@/components/dialogs'
+
+@Component({
+  components: { AddStaffNotationDialog }
+})
+export default class StaffNotation extends Vue {
+  private isAddingRegistrarsNotation = false
+  private isAddingRegistrarsOrder = false
+  private isAddingCourtOrder = false
+  private expand = false
+
+  /** Prop for the scrollbar offset to be added. */
+  @Prop() private addScrollbarOffset: string
+
+  showRegistrarsNotationDialog (): void {
+    this.isAddingRegistrarsNotation = true
+  }
+  hideRegistrarsNotationDialog (needReload: boolean): void {
+    this.isAddingRegistrarsNotation = false
+    this.close(needReload)
+  }
+  showRegistrarsOrderDialog (): void {
+    this.isAddingRegistrarsOrder = true
+  }
+  hideRegistrarsOrderDialog (needReload: boolean): void {
+    this.isAddingRegistrarsOrder = false
+    this.close(needReload)
+  }
+  showCourtOrderDialog (): void {
+    this.isAddingCourtOrder = true
+  }
+  hideCourtOrderDialog (needReload: boolean): void {
+    this.isAddingCourtOrder = false
+    this.close(needReload)
+  }
+
+  @Emit('close')
+  private close (needReload: boolean): void {
+    // Intentionally empty
```

Comment:
Who says this is a code smell without a body? We use `{}` in many places. I'm OK with that.